docs/tags/common-schema.md

```diff
@@ -24,6 +24,7 @@ Refer to this list of default fields for Logs Apps. The tags below are applicabl
 | error.type | A reserved field for errors |
 | span.id | Building block of a trace in distributed tracing |
 | trace.id | Building block of a trace in distributed tracing |
+| geo.ip | IP address defining the location associated with the log line |
```

Comment:
Does this make sense?  Does it make sense to add a field with a geo prefix?  That assumes the IP will be geodecoded.  Isn't there os.hostip?

Reply:
I think it makes sense. It is not about the host IP. Think Apache logs. You are not interested in the IP of your host, but the IP of client that made the request.

Reply:
Yes, the hostip is not the public IP address. The public IP will either be taken from the logs (e.g apache) or will be looked up inside Logagent and then it will be the public IP of the host. It makes sense to me.